config/environments/development.rb

```diff
@@ -1,39 +1,38 @@
-Accounts::Application.configure do
+Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb
 
   # In the development environment your application's code is reloaded on
   # every request. This slows down response time but is perfect for development
   # since you don't have to restart the web server when you make code changes.
   config.cache_classes = false
 
-  # Log error messages when you accidentally call methods on nil.
-  config.whiny_nils = true
+  # Do not eager load code on boot.
+  config.eager_load = true
```

Comment:
`eager_load = true` means we DO eager load code boot, right? (comment conflicts with code?)